site/monitoring-cluster-operator.md

```diff
@@ -0,0 +1,165 @@
+
+    <pre class='hljs lang-yaml'>
+    spec:
+      podMonitorNamespaceSelector:
+        any: 'true'
+    </pre>
+
+1. Patch the Prometheus Operator by running:
+
+    <pre class='hljs lang-bash'>
+    kubectl -n monitoring patch prometheuses k8s --type merge --patch "$(cat rabbitmq-prometheus-patch.yaml)"
+    </pre>
+
+1. Create a YAML file named `rabbitmq-podmonitor.yaml` on your local machine.
+
+1. Copy and paste the content below into `rabbitmq-podmonitor.yaml`, then save the file.
+
+    <pre class='hljs lang-yaml'>
+    apiVersion: monitoring.coreos.com/v1
+    kind: PodMonitor
+    metadata:
+      name: rabbitmq
+    spec:
+      podMetricsEndpoints:
+      - interval: 15s
+        port: prometheus
+      selector:
+        matchLabels:
+          app.kubernetes.io/component: rabbitmq
+      namespaceSelector:
+        any: true
+    </pre>
+
+    You now have the `PodMonitor` resource, which is needed to configure the automatic discovery of RabbitMQ
+    clusters.
+
+1. Apply the `PodMonitor` resource by running: `kubectl apply -f rabbitmq-podmonitor.yaml`
+
+`PodMonitor` can be created in any namespace, as long as the Prometheus Operator has permissions to find it.
+For more information about these permissions, see
+[Configure Permissions for the Prometheus Operator](#config-perm) below.
+
+
+### <a id='config-perm'></a> (Optional) Configure Permissions for the Prometheus Operator
+
+If your RabbitMQ clusters do not appear in Prometheus, you might need to configure permissions for the
+Prometheus Operator.
+
+For more information about permissions, see
+[RBAC](https://github.com/coreos/prometheus-operator/blob/master/Documentation/rbac.md) in GitHub.
+
+The following steps have been tested with a `kube-prometheus` deployment.
+To configure permissions for the Prometheus Operator:
+
+1. Create a YAML file somewhere on your local machine named `prometheus-roles.yaml`.
+
+1. Copy and paste the content below into `prometheus-roles.yaml` and save the file.
+
+    <pre class='hljs lang-yaml'>
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRole
+    metadata:
+      name: prometheus
+    rules:
+    - apiGroups: [""]
+      resources:
+      - nodes
+      - services
+      - endpoints
+      - pods
+      verbs: ["get", "list", "watch"]
+    - apiGroups: [""]
+      resources:
+      - configmaps
+      verbs: ["get"]
+    - nonResourceURLs: ["/metrics"]
+      verbs: ["get"]
+    \-\-\-\-
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRoleBinding
+    metadata:
+      name: prometheus
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: prometheus
+    subjects:
+    - kind: ServiceAccount
+      name: prometheus-k8s
+      namespace: monitoring
+    </pre>
+
+1. Apply the permissions listed in `prometheus-roles.yaml` by running: `kubectl apply -f prometheus-roles.yaml`
+
+## <a id='grafana'></a> Import Dashboards to Grafana
```

Comment:
This is a great chance to either improve our Prometheus and Grafana guide or link to it (or add a new section we can link to from here).

Reply:
I've updated the instructions in the Prometheus a Grafana guide to the list we have in here. I think that using grafana.com as a source to import the dashboards might be a more pleasant experience that copy pasting json from GitHub. Also added a link to the grafana dashboards section in the prometheus guide to refer to these instructions.